### PR TITLE
Promote/DO App Platform Deployment

### PR DESCRIPTION
**Pull request by black-bryan**
Promotion of the scim-example for DigitalOcean App Platform using the DO portal or DO & 1Password CLI tools from the "Beta" examples directory.

This process of deployment has been used by many customers and since this is a quick and easy low cost SCIM bridge deployment, it is time to move these deployment options out of Beta.
Link to DigitalOcean App Platform 1Password SCIM Bridge deployment using the DO portal
Promoted DigitalOcean App Platform using the DO command line tool and the 1Password CLI from the beta directory
Other changes:

- Introducing concept on main README.md and PREPARATION.md of Container-as-a-Service deployment, differentiating from Advanced and Marketplace deployments.
- Detailing on the PREPARATION.md, how DNS and certificate management are handled by the CaaS infrastructure automatically.

### CODE REVIEW

**Comment by black-bryan** [comment]
New PR from the work of PR280, as PR280 was not merging correctly.

File: do-app-platform-op-cli/README.md [hunk 1]
```diff
@@ -189,16 +191,16 @@ You can also access your SCIM bridge by visting the URL in your web browser. Sig
...
   Using Bash:
   ```sh
-  curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/google-workspace/op-scim-bridge-gw.yaml | op inject | doctl apps create --spec - --wait --upsert
+  curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/google-workspace/op-scim-bridge-gw.yaml | op inject | doctl apps create --spec - --wait --upsert
   ```
 
   Using Powershell:
   ```pwsh
-  Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/google-workspace/op-scim-bridge-gw.yaml` | op inject | doctl apps create --spec - --wait --upsert
+  Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/google-workspace/op-scim-bridge-gw.yaml` | op inject | doctl apps create --spec - --wait --upsert
```
Comment (ag-adampike) [inline do-app-platform-op-cli/README.md:203]: The URL should not be surrounded by backticks here:

```suggestion
  Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/google-workspace/op-scim-bridge-gw.yaml | op inject | doctl apps create --spec - --wait --upsert
```

File: do-app-platform-op-cli/README.md [hunk 2]
```diff
@@ -268,12 +270,12 @@ To revert to the default specification that is suitable for provisioning up to 1
...
 Using Bash:
 ```sh
-curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
 ```
 
 Using Powershell:
 ```pwsh
-Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml` | op inject | doctl apps create --spec - --wait --upsert
+Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml` | op inject | doctl apps create --spec - --wait --upsert
```
Comment (ag-adampike) [inline do-app-platform-op-cli/README.md:278]: Also here:

```suggestion
Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
```

File: do-app-platform-op-cli/README.md [hunk 3]
```diff
@@ -239,12 +241,12 @@ You can optionally propose the raw app spec template to verify the cost before d
...
 Using Bash:
 ```sh
-curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml | doctl apps propose --spec -
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | doctl apps propose --spec -
 ```
 
 Using Powershell: 
 ```pwsh
-Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml` | doctl apps propose --spec -
+Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml` | doctl apps propose --spec -
```
Comment (ag-adampike) [inline do-app-platform-op-cli/README.md:249]: Similar to above:

```suggestion
Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | doctl apps propose --spec -
```

**Review by ag-adampike — CHANGES_REQUESTED** [review]
Some spacing issues and extraneous backticks in the PowerShell commands, otherwise LGTM!

File: do-app-platform-op-cli/README.md [hunk 4]
```diff
@@ -113,13 +115,13 @@ For example:
...
 ```sh
-curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait
 ```
 
 Using Powershell:
 
 ```pwsh
-Invoke-RestMethod -Uri `https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml` | op inject | doctl apps create --spec - --wait
+Invoke-RestMethod -Uri `https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml` | op inject | doctl apps create --spec - --wait
```
Comment (black-bryan) [inline do-app-platform-op-cli/README.md:124]:
```suggestion
Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait
```

File: do-app-platform-op-cli/README.md [hunk 5]
```diff
@@ -211,12 +213,12 @@ If you chose your own name for the vault and items where you saved your SCIM bri
...
 Using Bash:
 ```sh
-curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
 ```
 
 Using Powershell:
 ```pwsh
-Invoke-RestMethod -Uri `https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml` | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
+Invoke-RestMethod -Uri `https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml` | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
```
Comment (black-bryan) [inline do-app-platform-op-cli/README.md:221]:
```suggestion
Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | VAULT="vault name" ITEM="item name" op inject | doctl apps create --spec - --wait
```

File: do-app-platform-op-cli/README.md [hunk 6]
```diff
@@ -225,12 +227,12 @@ The latest version of 1Password SCIM bridge is posted on our [Release Notes](htt
...
 Using Bash:
 ```sh
-curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
+curl -s https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
 ```
 
 Using Powershell:
 ```pwsh
-Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/beta/do-app-platform-op-cli/op-scim-bridge.yaml` | op inject | doctl apps create --spec - --wait --upsert
+Invoke-RestMethod -Uri`https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml` | op inject | doctl apps create --spec - --wait --upsert
```
Comment (black-bryan) [inline do-app-platform-op-cli/README.md:235]:
```suggestion
Invoke-RestMethod -Uri https://raw.githubusercontent.com/1Password/scim-examples/main/do-app-platform-op-cli/op-scim-bridge.yaml | op inject | doctl apps create --spec - --wait --upsert
```

**Review by black-bryan — COMMENTED** [review]
Additional backticks in Invoke-RestMethod -Uri path removed

**Review by ag-adampike — APPROVED** [review]
LGTM!

**Pull request merged**